### ISSUE TITLE
Document TTIR and TTNN ops

### ISSUE THREAD

**Issue by sdjordjevicTT** [tenstorrent/tt-mlir#401]
We need to improve documentation for our TTIR and TTNN dialects by providing clear descriptions and examples.

**Comment by staylorTT** [comment]
@sdjordjevicTT Is this still active? Should we get someone to work on this?

**Comment by sdjordjevicTT** [comment]
Yeah, this one is a placeholder to go over defined ops in TTIR and TTNN, and update the ops description, because based on that description we are generating our wiki docs:
https://docs.tenstorrent.com/tt-mlir/autogen/md/Dialect/TTIROp.html
https://docs.tenstorrent.com/tt-mlir/autogen/md/Dialect/TTNNOp.html

**Comment by skhatriTT** [comment]
Hey @sdjordjevicTT could you please provide our status on this one?

**Comment by sdjordjevicTT** [comment]
I will close this one; we already added a bunch of docs in our dialect ops definitions.
